nrf_802154/doc/CHANGELOG.rst

```diff
@@ -10,8 +10,8 @@ Changelog
 All notable changes to this project are documented in this file.
 See also :ref:`nrf_802154_limitations` for permanent limitations.
 
-Main branch - nRF 802.15.4 Radio Driver
-***************************************
+nRF Connect SDK v3.0.0 - nRF 802.15.4 Radio Driver
+**************************************************
 
 Added
 =====
```

Comment:
This PR will introduce new changes here: https://github.com/nrfconnect/sdk-nrfxlib/pull/1726